ddtrace/internal/telemetry/metrics.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import abc
+import time
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import six
+from typing_extensions import Literal
+
+from ..hostname import get_hostname
+
+
+MetricType = Literal["count", "gauge", "rate"]
+MetricTagType = Dict[str, str]
+
+
+class Metric(six.with_metaclass(abc.ABCMeta)):
+    """
+    stores metrics which will be sent to the Telemetry Intake metrics to the Datadog Instrumentation Telemetry Org
+    """
+
+    HOST_NAME = get_hostname()
+    metric_type = ""
+
+    def __init__(self, namespace, name, tags, common, interval=None):
+        # type: (str, str, MetricTagType, bool, Optional[float]) -> None
+        """
+        name: metric name
```

Comment:
This comment doesn't add any information one can't get from the variable name. I'd probably remove it.